.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.yml]
+indent_size = 2
```

Comment:
This is the indentation as observed in [`.travis.yml`](https://github.com/fastly/fastly-php/blob/master/.travis.yml). [`phpspec/yml`](https://github.com/fastly/fastly-php/blob/master/phpspec.yml), however, is indented with `4` spaces. Was actually going to take a look at it, then noticed the indentation. Hence, this PR.